### ISSUE TITLE
FR: Support for sending notification to multiple devices

### ISSUE THREAD

**Issue by adihat** [firebase/firebase-admin-python#277]
Currently there is support to send notifications to specific devices as documented [here][1].

There is support for sending notifications to multiple devices in Node JS SDK and Java SDK.

Example:
```javascript
const registrationTokens = [
  'YOUR_REGISTRATION_TOKEN_1',
  // …
  'YOUR_REGISTRATION_TOKEN_N',
];

const message = {
  data: {score: '850', time: '2:45'},
  tokens: registrationTokens,
}

admin.messaging().sendMulticast(message)
  .then((response) => {
    console.log(response.successCount + ' messages were sent successfully');
```

This operation uses the `sendAll()` API under the hood and return value is a `BatchResponse`.

It would be good if there was something similar in python sdk:
```python
registration_tokens = [
       'YOUR_REGISTRATION_TOKEN_1',
       // …
      'YOUR_REGISTRATION_TOKEN_N',
]
message = messaging.Message(
    data={
        'score': '850',
        'time': '2:45',
    },
    tokens=registration_tokens,
)
response = messaging.send_multicast(message)
print(f"{response['successCount']} messages were sent successfully")
```

[1]: https://firebase.google.com/docs/cloud-messaging/send-message

**Comment by aaugustin** [comment]
Unless I missed something, this issue was fixed by #283 and can be closed.

It would be nice to have a PyPI release with this feature :-) Thank you :-)

**Comment by hiranya911** [comment]
Release on its way.
